CellFabric/Cell_Fabric_FinFET__Mock/Align_primitives.py

```diff
@@ -18,22 +17,13 @@ def main( args):
     gu = gate + 2*gateDummy
     y_cells = args.Ycells
     x_cells = 2*args.Xcells
+    # Why do we have this condition? x_cells == 3 allows CC
     pattern = 2 if x_cells%4 != 0 else args.pattern ### CC is not possible; default is interdigitated
```

Comment:
Why do we have this condition? x_cells == 3 allows CC for example

Reply:
I tried with two different devices (each having 3 unit cells) but couldn't implement without the use of dummy transistors. Is it possible without dummies? Currently, I'm working on an algorithm which could be used with dummies.

Reply:
Not going to address this as part of this pull request.